### PR TITLE
customer docker spawner class

### PR DESCRIPTION
**Pull request by Tianhao-Gu**
Copied directly from `custom_spawner.py` which will be deleted once docker spawner is setup.

### CODE REVIEW

File: src/jupyterhub_config/custom_docker_spawner.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
+import os
+import venv
+from pathlib import Path
+
+from dockerspawner import DockerSpawner
+
+
+class CustomDockerSpawner(DockerSpawner):
```
Comment (Tianhao-Gu) [inline src/jupyterhub_config/custom_docker_spawner.py:8]: it now inherits from the DockerSpawner module. All other funcs are copied from `custom_spawner`

**Comment by MrCreosote** [comment]
To be clear I've already reviewed everything in the file here; the only change is the superclass?

**Comment by Tianhao-Gu** [comment]
> To be clear I've already reviewed everything in the file here; the only change is the superclass?

yea. Directly copied and pasted from  previous `custom_spawner` class.

**Review by MrCreosote — APPROVED** [review]
Approving w/o further review based on the above

**Pull request merged**